_emails/us/california/santa-monica.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Santa Monica
+permalink: "/santamonica"
+name: Letter to City Managers and Council Members
+city: Santa Monica
+state: CA
+recipients:
+- info@santamonica.gov
+- rick.cole@smgov.net
+- manager@smgov.net
+- Katie.Lichtig@smgov.net
+- Anuj.Gupta@smgov.net
+- Christopher.J.Smith@smgov.net
+- debbie.lee@smgov.net
+- Erin.Taylor@smgov.net
+- Constance.Farrell@smgov.net
+- russ.maloney@smgov.net
+- Tim.Dodd@smgov.net
+- Lindsay.Barker@smgov.net
+- julie.rusk@smgov.net
+- alisa.orduna@smgov.net
+- council@smgov.net
+- gleam.davis@smgov.net
+- terry.oday@smgov.net
+- anamaria.jara@smgov.net
+- kevin.mckeown@smgov.net
+- sue.himmelrich@smgov.net
+- greg.morena@smgov.net
+- ted.winterer@smgov.net
+subject:
+body: |-
+  Dear Santa Monica City Managers and Council Members:
+
+  My name is [YOUR NAME] and I am a resident of [YOUR NEIGHBORHOOD]. I am writing to demand the allocation of resources to prioritize the safety and well-being of the city, and away from policing.
+
+  In the fiscal year of 2019-2020, Santa Monica's Police Departments allocated budget have escalated to a total of $97 million. Their budget has increased by 5.7% ($5 million) since the fiscal year of 2018-2019. This funding is in sharp contrast to the decrease of 19.1% ($2 million) spent towards affordable housing and economic development, and the meager increase of 3.1% ($1 million) in community and cultural services.
```

Comment:
I missed this in the content review, should be Santa Monica Police Departments' allocated budget has escalated (note the "has" and the apostrophe)

Reply:
Fixed this particular issue.

Reply:
On that note, shouldn't it be "I am writing to demand the _reallocation_" instead of "I am writing to demand the allocation"?

Reply:
Good catch -- that makes sense to me.

Reply:
Fixed. Resolving this conversation.